modules/navier_stokes/doc/content/source/fvkernels/PINSFVMomentumFriction.md

```diff
@@ -2,19 +2,74 @@
 
 This kernel adds the friction term to the porous media Navier Stokes momentum
 equations. This kernel must be used with the canonical PINSFV variable set,
-e.g. pressure and superficial velocity.  A variety of models are supported for
-the friction force:
+e.g. pressure and superficial velocity. This kernel supports Darcy and
+Forchheimer friction models:
 
 Darcy drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_d
+\label{darcy}
+\epsilon F_i = - f_i \rho \epsilon \frac{v_{D,i}}{\epsilon} = -f_i \rho v_{D,i}
 \end{equation}
 Forchheimer drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_d
+\label{forchheimer}
+\epsilon F_i = - f_i \rho \epsilon \frac{v_{D,i}}{\epsilon}\frac{|v_D|}{\epsilon} = - f_i \rho v_{D,i} \frac{|v_D|}{\epsilon}
+\end{equation}
+where $F_i$ is the i-th component of the friction force (denoted by $\mathbf{F_f}$ in [!eqref](pinsfv.md#eq:pinsfv_mom)), $f_i$ the friction factor, which may be anisotropic,
+$\epsilon$ the porosity and $\rho$ the fluid density and $v_{D,i}$ the i-th
+component of the fluid
+superficial velocity. We have used a negative sign to match the notation used in
+[!eqref](pinsfv.md#eq:pinsfv_mom) where the friction force is on the
+right-hand-side of the equation. When moved to the left-hand side, which is done
+when setting up a Newton scheme, the term becomes positive which is what is
+shown in the source code itself.
+Darcy and Forchheimer terms represent fundamentally different friction
+effects. Darcy is meant to represent viscous effects and as shown in [darcy]
+has a linear dependence on the fluid velocity. Meanwhile, Forchheimer is meant to
+represent inertial effects and as shown in [forchheimer] has a quadratic dependence on velocity.
+
+## Computation of friction factors and pre-factors id=friction_example
+
+To outline how friction factors for Darcy and Forchheimer may be calculated,
+let's consider a specific example. We'll draw from the Ergun equation, which is
+outlined [here](https://en.wikipedia.org/wiki/Ergun_equation). Let's consider
+the form:
+
+\begin{equation}
+\Delta p = \frac{150\mu L}{d_p^2} \frac{(1-\epsilon)^2}{\epsilon^3} v_D + \frac{1.75 L \rho}{d_p} \frac{(1-\epsilon)}{\epsilon^3} |v_D| v_D
+\end{equation}
+
```

Comment:
to make it extra clear, I would add a 

We define the Darcy $f_D$ and Forchheimer $f_F$ friction factor from the pressure drop as:
```
\Delta p = ... f_D U ... + f_F ... U^2
 ```
 
 then it s extra clear

Reply:
I'm hoping it's pretty clear now, but if you still want more clarity, let me know